crates/utils/src/settings.rs

```diff
@@ -56,7 +97,16 @@ pub struct EmailConfig {
 #[derive(Debug, Deserialize, Clone)]
 pub struct CaptchaConfig {
   pub enabled: bool,
-  pub difficulty: String, // easy, medium, or hard
+  pub difficulty: String,
+}
+
+impl Default for CaptchaConfig {
+  fn default() -> Self {
+    Self {
+      enabled: true,
```

Comment:
Side note, but I'd like to get rid of these `enabled` fields. There's no point in having them if that config block is optional. 

IE if I'm including a captcha or federation config, I already know that I want it `enabled`.

Reply:
It makes sense if we remove env vars, but I would like to keep them. With env vars, it can make sense to set `LEMMY_CAPTCHA_ENABLED=false`, so its definitely disabled (without knowing whats in the config file).